pkg/harvester/store/harvester-store/index.ts

```diff
@@ -1,13 +1,16 @@
 //@ts-nocheck
 import { CoreStoreSpecifics, CoreStoreConfig } from '@shell/core/types';
 
+import SteveSchema from '@shell/models/steve-schema';
 import { SteveFactory, steveStoreInit } from '@shell/plugins/steve/index';
 import { PRODUCT_NAME } from '../../config/harvester';
 import getters from './getters';
 import mutations from './mutations';
 import actions from './actions';
 
-const harvesterFactory = (): CoreStoreSpecifics => {
+const harvesterFactory = (config): CoreStoreSpecifics => {
+  SteveSchema.reset(config.namespace);
+
   const steveFactory = SteveFactory(null, null);
```

Comment:
this doesn't look right. the first arg is namespace which i think is important?

Reply:
The fix from https://github.com/harvester/harvester-ui-extension/pull/36/files looks a bit scary too. That will overwrite the core schema for all stores without it being in the `harvester` folder.

Could be this line breaking it, given the namespace is missing it's not using <namespace>/schema?

Reply:
I tested it adding the `config` param to this code, but I still have the bug. I'm going to close this PR since it's not actually fixing the issue.